fake_switches/netconf/capabilities.py

```diff
@@ -62,6 +62,7 @@ def edit_config(self, request):
 
     def commit(self, _):
         self.datastore.commit_candidate()
+        self.datastore.configurations.get('candidate').commit()
```

Comment:
Here it feels natural, I read that I commit on the 'candidate' provided by the datastore.configurations, but it seams to be the only case.